clarifai/rest/client.py

```
@@ -97,7 +94,7 @@ def tag_urls(self, urls, model_name=DEFAULT_TAG_MODEL, model_id=None):
     """
 
     # validate input
-    if not isinstance(urls, list) or (len(urls) > 1 and not isinstance(urls[0], basestring)):
+    if not isinstance(urls, list) or (len(urls) > 1 and not isinstance(urls[0], str)):
       raise UserError('urls must be a list of string urls')
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)